samples/net/throughput_server/README.rst

```diff
@@ -0,0 +1,60 @@
+.. _throughput-server-sample:
+
+Throughput Server
+#################
+
+Overview
+********
+
+The throughput-server sample application for Zephyr implements an UDP server
+that complements the throughput-client sample application:
+the throughput-server listens for incoming IPv4 or IPv6 packets (sent by the
+throughput-client) and drops the packets or optional sends the packets back.
+Note that currently this sample is only meant to be used with ethernet bearer.
+
+The source code for this sample application can be found at:
+:file:`samples/net/throughput_server`.
+
+Building and Running
+********************
+
+There are multiple ways to use this application. Do not try to use this
+sample application in QEMU. The ethernet connection in QEMU is done using
+serial port with SLIP so it is very slow.
+
+There are configuration files for different boards and setups in the
+throughput-server directory:
+
+- :file:`prj_frdm_k64f.conf`
+  Use this for FRDM-K64F board with built-in ethernet.
+
+Build throughput-server sample application like this:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/throughput_server
+   :board: <board to use>
+   :conf: <config file to use>
+   :goals: build
+   :compact:
+
+Make can select the default configuration file based on the BOARD you've
+specified automatically so you might not always need to mention it.
+
+Running throughput-client in Linux Host
+=======================================
+
+There is one useful testing scenario that can be used with Linux host.
+Here throughput-server is run in real HW and throughput-client is run in
```

Comment:
Here, throughput-server is run on real HW and throughput-client
     is run on a Linux host.